src/rpc/rawtransaction.cpp

```diff
@@ -1121,7 +1122,9 @@ UniValue decodepsbt(const JSONRPCRequest& request)
             ScriptToUniv(txout.scriptPubKey, o, true);
             out.pushKV("scriptPubKey", o);
             in.pushKV("witness_utxo", out);
-        } else if (input.non_witness_utxo) {
```

Comment:
In commit "rpc: show both UTXOs in decodepsbt" (72f6bec1da198764d4648a10a61c485e7ab65e9e)

Note: Bug here is fixed in #19517 (changing `else if` to `if` can allow `total_in` to be incremented twice with the same amount)